src/static/main.css

```diff
@@ -0,0 +1,32 @@
+header {
```

Comment:
hm... you wanna delete `header`?

Reply:
Its filled in now. It was only empty cus I was gonna do all the css and styles for UI in next commit. If you look now its a rather sexy interface.